samples/examples/BinaryClasification_Titanic/BinaryClasification_TitanicSurvivalPrediction/Program.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+namespace BinaryClasification_TitanicSurvivalPrediction
+{
+
+    internal static class Program
+    {
+        private static string AppPath => Path.GetDirectoryName(Environment.GetCommandLineArgs()[0]);
+        private static string TrainDataPath => Path.Combine(AppPath, "datasets", "titanic-train.csv");
+        private static string TestDataPath => Path.Combine(AppPath, "datasets", "titanic-test.csv");
+        private static string ModelPath => Path.Combine(AppPath, "TitanicModel.zip");
+
+        private static async Task Main(string[] args)
+        {
+            // STEP 1: Create a model
+            var model = await TrainAsync();
+
+            // STEP2: Test accuracy
+            Evaluate(model);
+
+            // STEP 3: Make a prediction
+            var prediction = model.Predict(TestTitanicData.Passenger);
```

Comment:
I have a suggestion. How about we remove one passenger from the test dataset, get his/her parameters into the TestTitanicData.Passenger and do a prediction on it. That way our model would not be exposed to the data before which is more clear experiment.

Reply:
FIxed in: 027b4be